website/app/templates/head.hbs

```diff
@@ -0,0 +1,90 @@
+{{!
+
+Per https://github.com/shipshapecode/ember-meta#defining-your-own-headhbs this is a place where we can customize what information is injected by the ember-meta addon
+
+We've made some small customizations to simplify this compared to the default template
+
+}}
+
+{{! Begin General }}
+
+{{! TODO: Determine how to avoid this conflict between ember-meta and ember-page-title where they both set a <title> which causes the latter addon to throw an exception }}
+{{!-- {{#if this.model.title}}
+  <title>{{this.model.title}}</title>
+{{/if}} --}}
+
+{{#if this.model.description}}
+  <meta name="description" content={{this.model.description}} />
+{{/if}}
+
+<meta name="referrer" content="unsafe-url" />
+
+{{#if this.model.canonical}}
+  <link rel="canonical" href={{this.model.canonical}} />
+{{else if this.model.url}}
+  <link rel="canonical" href={{this.model.url}} />
+{{/if}}
+{{! End General }}
+
+{{! Begin Article }}
+{{!-- {{#if this.model.date}}
+  <meta property="article:published_time" content={{this.model.date}} />
+{{/if}}
+
+{{#each this.model.tags as |tag|}}
+  <meta property="article:tag" content={{tag}} />
+{{/each}} --}}
+{{! End Article }}
+
+{{! Begin opengraph (Facebook, Slack, etc) }}
+{{#if this.model.siteName}}
+  <meta property="og:site_name" content={{this.model.siteName}} />
+{{/if}}
+
+{{#if (or this.model.articleTitle this.model.title)}}
+  <meta property="og:title" content={{if this.model.articleTitle this.model.articleTitle this.model.title}} />
+{{/if}}
```

Comment:
For consistency with lines 22-26 (and better readability)

```suggestion
{{#if this.model.articleTitle}}
  <meta property="og:title" content={{this.model.articleTitle}} />
{{else if this.model.title}}
  <meta property="og:title" content={{this.model.title}} />
{{/if}}
```

Reply:
Also, where is this `articleTitle ` value coming from? I can't see it in the code (apart from here) so my understanding is that it's never a real case

Reply:
in our case this logic relies on `this.model.title` existing, so we could update it to remove the `articleTitle` references completely.

Reply:
Yes, that's what I meant